cmd/expiration-mailer/main.go

```diff
@@ -485,11 +500,21 @@ type Config struct {
 		DB cmd.DBConfig
 		cmd.SMTPConfig
 
-		From    string
+		// The "From" address for reminder messages.
```

Comment:
Nit, here and below you're not follwing the go-doc comment style for struct fields.